python/phylanx/ast/physl.py

```diff
@@ -772,6 +773,20 @@ def _NestedSubscript(node):
         #         return [op, (value, slice_)]
         return [op, (value, slice_)]
 
+    def _With(self, node):
+        if 0 < len(node.items) and type(node.items[0]) == ast.withitem:
```

Comment:
This should be handled in the `_Withitem` method.

Reply:
Not sure how that can be done. The information needed for the block is not completely contained under ast.withitem.